src/Module/Admin/Storage.php

```diff
@@ -40,7 +40,7 @@ public static function post(array $parameters = [])
 		$storagebackend = Strings::escapeTags(trim($parameters['name'] ?? ''));
 
 		try {
-			/** @var IStorageConfiguration|false $newStorageConfig */
+			/** @var \Friendica\Core\Storage\Capability\ICanConfigureStorage|false $newStorageConfig */
```

Comment:
Extraneous absolute name:
```suggestion
			/** @var ICanConfigureStorage|false $newStorageConfig */
```